src/CMakeLists.txt

```diff
@@ -87,6 +87,7 @@ set_target_properties(gpuarray PROPERTIES
   COMPILE_FLAGS "-DGPUARRAY_BUILDING_DLL -DGPUARRAY_SHARED"
   INSTALL_NAME_DIR ${CMAKE_INSTALL_PREFIX}/lib
   MACOSX_RPATH OFF
+  VERSION 0.0
```

Comment:
How we can get access to that version in Theano? Does this version if an interface change or any code version upgrade? I think we need to keep that information available in the version in some way.